topomodelx/nn/hypergraph/hmpnn.py

```diff
@@ -35,32 +35,34 @@ class HMPNN(torch.nn.Module):
 
     def __init__(
         self,
-        in_features,
-        hidden_features,
-        num_classes,
-        n_layer=2,
+        in_channels,
+        hidden_channels,
+        n_layers=2,
         adjacency_dropout_rate=0.7,
         regular_dropout_rate=0.5,
     ):
         super().__init__()
-        hidden_features = (in_features,) + hidden_features
-        self.to_hidden_linear = torch.nn.Sequential(
-            *[
-                torch.nn.Linear(hidden_features[i], hidden_features[i + 1])
-                for i in range(len(hidden_features) - 1)
-            ]
-        )
+
+        self.linear_node = torch.nn.Linear(in_channels, hidden_channels)
+        self.linear_edge = torch.nn.Linear(in_channels, hidden_channels)
+        # self.to_hidden_linear = torch.nn.Sequential(
```

Comment:
No left over comments on main branch: delete (keep it on your own branch if you want)